runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -280,6 +280,7 @@ private ClueScroll findClueScroll()
 			String text = clueScrollText.getText()
 					.replaceAll("-<br>", "-")
 					.replaceAll("<br>", " ")
+					.replaceAll("<col=[^>]*>", "")
```

Comment:
We have a removeTags method in the Text class, I don't know if you maybe want to use that instead.